python/tchannel/tornado/util.py

```diff
@@ -52,3 +53,21 @@ def print_arg(request, index):
                 print chunk
     except Exception:
         pass
+
```

Comment:
Dude. Rename this file to `json.py` so we don't end up dumping a bunch of un-related functions in it.